docs/source/02_installation.md

```diff
@@ -1,4 +1,4 @@
-# Building ADAM from Source {#build-from-source}
+# Building ADAM from Source
```

Comment:
@fnothaft @gunjanbaid @devin-petersohn send me your readthedocs user names and I'll add you as maintainers with admin access.  There is an admin UI with the hooks and other configurations.